trade-finance-manager-ui/component-tests/utilisation-reports/_macros/payment-details-table-row.component-test.ts

```diff
@@ -26,14 +26,17 @@ describe(component, () => {
     feeRecordPaymentGroupStatus: FEE_RECORD_STATUS.TO_DO,
     payment: aPaymentDetailsPayment(),
     feeRecords: [{ facilityId: '12345678', exporter: 'Test exporter' }],
-    reconciledBy: undefined,
-    dateReconciled: undefined,
+    reconciledBy: '-',
+    dateReconciled: {
+      formattedDateReconciled: '-',
+      dataSortValue: 0,
+    },
   });
 
   const getWrapper = ({ reportId, paymentDetailsRow, userCanEdit }: { reportId?: number; paymentDetailsRow: PaymentDetailsTableRow; userCanEdit?: boolean }) =>
     render({ reportId, paymentDetails: paymentDetailsRow, userCanEdit });
 
-  it('renders the payment reference and amount and fee record facility ID and exporter', () => {
+  it('renders the payment reference and amount, the fee record facility ID and exporter and the date reconciled and reconciled by user', () => {
```

Comment:
Would be nice to split up since there's a lot of "ands" here, e.g 1x test for each:

```ts
it renders the payment reference
it renders amount
it renders the fee record facility ID
it renders the exporter
it renders the date reconciled
it renders the reconciled by user
```